Source/bmalloc/libpas/src/libpas/pas_status_reporter.c

```diff
@@ -1135,7 +1138,27 @@ void pas_status_reporter_dump_everything(pas_stream* stream)
     pas_status_reporter_dump_expendable_memories(stream);
 }
 
+void pas_sleep(unsigned microseconds);
+
+#if PAS_OS(WINDOWS)
+#define R_NULL 0
+
+void pas_sleep(unsigned microseconds)
+{
+    Sleep(microseconds);
+}
+
+unsigned status_reporter_thread_main(void* arg)
+#else
+#define R_NULL NULL
+
+void pas_sleep(unsigned microseconds)
+{
+    usleep(microseconds);
+}
+
 static void* status_reporter_thread_main(void* arg)
```

Comment:
This code looks very weird. You should not involve `pas_sleep` definition and this function's signature in one if-def.